src/libs/state.ts

```diff
@@ -149,20 +150,17 @@ export const getTerraUser = (): TerraUser => authStore.get().terraUser;
 
 export const getSessionId = () => authStore.get().sessionId;
 
-export type OidcConfig = {
-  authorityEndpoint?: string;
-  clientId?: string;
-};
-
 export type OidcState = {
   authContext: AuthContextProps | undefined;
+  authTokenState: AuthTokenState | undefined;
```

Comment:
If we're wrapping the entire app in `OAuth2.initializeClientId`, are these ever undefined?